web/src/main/java/nl/tehdreamteam/se42/web/command/impl/ExitApplicationCommand.java

```diff
@@ -0,0 +1,37 @@
+package nl.tehdreamteam.se42.web.command.impl;
+
+import nl.tehdreamteam.se42.web.ServiceContainer;
+import nl.tehdreamteam.se42.web.command.Command;
+import nl.tehdreamteam.se42.web.command.CommandHandler;
+
+public class ExitApplicationCommand extends Command {
+
+    private final ServiceContainer container;
+
+    public ExitApplicationCommand(ServiceContainer container) {
+        this.container = container;
+    }
+
+    @Override
+    public void execute(CommandHandler handler, String line, String[] args) {
+        container.stopAll();
+
+        System.exit(0);
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Remove this call to "exit" or ensure it is really required. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonarqube.tehmessiah.nl/coding_rules#rule_key=squid%3AS1147)